.github/workflows/macOS.yml

```diff
@@ -8,6 +8,35 @@ jobs:
       CC: gcc-11
       CXX: g++-11
```

Comment:
No need to set CXX variable. On MacOS I suggest we test this library using native compiler (clang).

Reply:
OK